ReactWindows/ReactNative/Views/DatePicker/ReactCalendarDatePickerShadowNode.cs

```diff
@@ -0,0 +1,24 @@
+using Facebook.Yoga;
+using ReactNative.UIManager;
+
+namespace ReactNative.Views.DatePicker
+{
+    /// <summary>
+    /// The shadow node implementation for <see cref="ReactNative.Views.DatePicker.ReactCalendarDatePickerManager" /> views.
+    /// </summary>
+    public class ReactCalendarDatePickerShadowNode : LayoutShadowNode
+    {
+        /// <summary>
+        /// Instantiates the <see cref="ReactCalendarDatePickerShadowNode"/>.
+        /// </summary>
+        public ReactCalendarDatePickerShadowNode()
+        {
+            MeasureFunction = MeasurePicker;
+        }
+
+        private static YogaSize MeasurePicker(YogaNode node, float width, YogaMeasureMode widthMode, float height, YogaMeasureMode heightMode)
+        {
+            return MeasureOutput.Make(width, 32f);
```

Comment:
I wonder if we can pull the `32f` value from somewhere else?

Reply:
(Same would go for the other const values supplied for height). I suspect this value would vary based on user accessibility settings.

Reply:
I agree. The only issue that I am having is with handling a default height. If it is not passed as a style, the height value is 900, which consumes the entirety of the screen. However, I am following the same logic as that of `ReactProgressBarShadowNode.cs`:

```csharp
        private static YogaSize MeasureProgressBar(YogaNode node, float width, YogaMeasureMode widthMode, float height, YogaMeasureMode heightMode)
        {
            var adjustedHeight = YogaConstants.IsUndefined(height) ? 4f : height; 
            return MeasureOutput.Make(width, adjustedHeight);
        }
```

Any ideas?